ompi/win/win.h

```diff
@@ -152,27 +151,27 @@ OMPI_DECLSPEC int ompi_win_group(ompi_win_t *win, ompi_group_t **group);
 /* Note that the defintion of an "invalid" window is closely related
    to the defintion of an "invalid" communicator.  See a big comment
    in ompi/communicator/communicator.h about this. */
-static inline int ompi_win_invalid(ompi_win_t *win) {
-    if (NULL == win ||
-        MPI_WIN_NULL == win ||
-        (OMPI_WIN_INVALID & win->w_flags) ||
+static inline int ompi_win_invalid(ompi_win_t *win)
+{
+    if (NULL == win || MPI_WIN_NULL == win || (OMPI_WIN_INVALID & win->w_flags) ||
         (OMPI_WIN_FREED & win->w_flags)) {
         return true;
     } else {
         return false;
     }
 }
 
-static inline int ompi_win_peer_invalid(ompi_win_t *win, int peer) {
-    if (win->w_group->grp_proc_count <= peer || peer < 0) return true;
+static inline int ompi_win_peer_invalid(ompi_win_t *win, int peer)
+{
+    if (win->w_group->grp_proc_count <= peer || peer < 0)
+        return true;
     return false;
 }
 
-static inline int ompi_win_rank(ompi_win_t *win) {
-    return win->w_group->grp_my_rank;
-}
+static inline int ompi_win_rank(ompi_win_t *win) { return win->w_group->grp_my_rank; }
```

Comment:
I don't think we should be making this type of change.  Functions should really be formatted the same every time, to reduce cognitive load.

Reply:
Ok, options are:

SFS_None (in configuration: None) Never merge functions into a single line.
SFS_InlineOnly (in configuration: InlineOnly) Only merge functions defined inside a class. Same as “inline”, except it does not implies “empty”: i.e. top level empty functions are not merged either.
SFS_Empty (in configuration: Empty) Only merge empty functions.
SFS_All (in configuration: All) Merge all functions fitting on a single line.

Right now it is All but consistency suggest we should use None. WDYT?

Reply:
Yeah, that sounds better to me.

Reply:
I think we have some empty functions and I have less care there, so I'd vote either none or empty.

Reply:
Fixed. I will open a PR with the .clang-format change once I have resolved the conditional formatting.